pkgs/unified_analytics/test/survey_handler_test.dart

```diff
@@ -117,6 +119,7 @@ void main() {
 		"description": "xxxxxxx",
 		"snoozeForMinutes": "10",
 		"samplingRate": "1.0",
+    "excludeDashTools": [],
```

Comment:
The indentation in several of these tests seems off. It doesn't change the semantics of the test, but makes them a little harder to read. Do some of them use tabs while others use strings?